docs/roadmap.md

```diff
@@ -0,0 +1,83 @@
+# Roadmap
```

Comment:
This is just a general nit, but I wonder if there's value in just adding sections with our specific names/list of pet projects. Literally just a heading named `## Austin` or `## Yuya` for example. I think I have at least 2 things I'd like to put down in here for the forseeable future, but it's probably useful to also know they're my bailiwick so people can ask or contact me about it.

Reply:
IMO: Get buy-in from a few others and add your ideas to a "Undecided" section instead.

Related: I meant this roadmap to have only items I felt like we have consensus on. If there are things here that seem controversial, we should remove them or move them to the "Undecided" section.

Reply:
I don't think adding the people responsible to it makes sense, at these are the common project goals. IMO, if you want a feature and responsibility for it, create an issue and assign to you. 

Because adding yourself to some long-term communicated goal can badly missfire.